x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -654,10 +716,44 @@ export class ExplorerChartDistribution extends React.Component {
     const isLoading = seriesConfig.loading;
 
     return (
-      <div className="ml-explorer-chart" ref={this.setRef.bind(this)}>
-        {isLoading && <LoadingIndicator height={CONTENT_WRAPPER_HEIGHT} />}
-        {!isLoading && <div className="content-wrapper" />}
-      </div>
+      <>
+        <RuleEditorFlyout
+          setShowFunction={this.setShowRuleEditorFlyoutFunction}
+          unsetShowFunction={this.unsetShowRuleEditorFlyoutFunction}
+        />
+        {this.state.popoverData !== null && (
+          <div
+            style={{
+              position: 'absolute',
+              marginLeft: this.state.popoverCoords[0],
+              marginTop: this.state.popoverCoords[1],
+            }}
+          >
+            <EuiPopover
+              isOpen={true}
+              closePopover={() => this.closePopover()}
+              panelPaddingSize="none"
+              anchorPosition="upLeft"
+            >
+              <LinksMenuUI
+                anomaly={this.state.popoverData}
+                bounds={this.props.bounds}
+                showMapsLink={false}
+                showViewSeriesLink={true}
```

Comment:
The 'View series' link isn't working for me. I see this error:

<img width="1287" alt="Screenshot 2024-06-21 at 14 42 51" src="https://github.com/elastic/kibana/assets/7405507/e03353c5-7146-4fac-83ab-661981a1b7ed">

Reply:
Fixed in [db0bd6f](https://github.com/elastic/kibana/pull/186587/commits/db0bd6f8adb5fc953f85b1b714a4732ee0e2d62c).

Reply:
Time range is being passed correctly. But now I'm seeing that the partition field is often not set correctly in the SMV. Happens from the Anomaly Explorer or the embeddable.

Reply:
Are you still seeing this, @peteharverson? I wasn't able to reproduce when testing but can create more complex jobs if needed.

Reply:
I was finally able to reproduce this. It happened when there were multiple anomalies for different entities happening at the same time. I added a filter so we make sure only matching anomalies are considered for the popover. Adjusted the unit tests with new mock data for this case in [8e8185b](https://github.com/elastic/kibana/pull/186587/commits/8e8185b8f27c811688ebfa23ed17265a1f751660).

Reply:
Confirmed that adding in the filter fixes this for the test case that was occasionally opening the wrong series previously.